PUBLISHING.md

```diff
@@ -22,10 +23,29 @@ This will:
 - Push this tag to the remote
 - Trigger the new version publish
 
-## Rollback a new Release
+You can use the "Release latest-rc" manual github action to publish the last commit on main as the `latest-rc` version
 
+## Update tag version
+
+```sh
+$ yarn npm tag add sfdx-git-delta@<version-source> <version-target> 
+```
+
+It will set the version-source release channel to the version-target.
+
+Per example if you need to rollback `latest` to `v5.0.0`:
+```sh
+$ yarn npm tag add sfdx-git-delta@v5.0.0 latest
+```
+
+Per example if you want `latest` to be `latest-rc`:
```

Comment:
```suggestion
**Update `latest` to be `latest-rc`:**
This will update the SGD version most users get. The `latest` version is the one installed by default with the `sfdx plugins:install sfdx-git-delta` command.
```